lib/components/atoms/Image/Image.js

```diff
@@ -57,3 +57,5 @@ export default styled(Image)`
   ${styles};
 `;
 export { Image as ImageVanilla };
+
+export type { Props as ImageProps };
```

Comment:
They are already exported through type file. No need for an export here. Use that file to import.

Reply:
This is in same file, Refactoring type  in types file.